### DISCUSSION TITLE
Translation when data can not be displayed

### DISCUSSION THREAD

**Discussion by WohnungNDS** [open-sdg/open-sdg#1458] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
some combination of disaggregations can not be displayed. Can I translate this message?

![image](https://user-images.githubusercontent.com/67795376/140060284-18736a91-01b3-4d7d-9d00-a5207645c507.png)

[Indicator 1.3.1](https://wohnungnds.github.io/IM_Site/1-3-1/)

click on the radiobutton "in 1000" the preselected data cannot be displayed. I dont want do have another preselection for "in 1000" but I may give a hint in the message, that users schould try an other combination of disaggregation

**Comment by LucyGwilliamAdmin** [comment, marked as answer]
@WohnungNDS It does seem that the text is translated: https://github.com/open-sdg/sdg-translations/blob/1.7.0-dev/translations/de/indicator.yml#L4-L5 and that it should be getting translated.

I think you will need to add it to your local translations and then it should work for you.

**Comment by WohnungNDS** [comment]
@LucyGwilliamAdmin Thank You! .... I have added it to my translations... I think it will work, but i cannot test it because of #1422

**Comment by WohnungNDS** [comment]
Works fine" Thank you @LucyGwilliamAdmin 

![image](https://user-images.githubusercontent.com/67795376/141518606-9b013e28-e237-4e48-a400-6f6a2700f974.png)
